src/main/haskell/kore/src/Kore/Builtin/List.hs

```diff
@@ -457,17 +457,17 @@ unify
         -> Seq p
         -> p
         -> m (expanded, proof)
-    unifyFramedRight resultSort list1 symConcat prefix2 frame2
```

Comment:
Could you remove the unused argument in all the framing variable unification cases?